plasmapy/diagnostics/thomson.py

```diff
@@ -63,6 +63,7 @@ def spectral_density_lite(
     probe_vec: np.ndarray,
     scatter_vec: np.ndarray,
     instr_func_arr: Optional[np.ndarray] = None,
+    notches: Optional[np.ndarray] = None,
 ) -> tuple[Union[np.floating, np.ndarray], np.ndarray]:
     r"""
```

Comment:
You need to add this keyword to the docstring. What is the shape of the required array? Can it include multiple notches?